docs/docs/platforms/OPENSHIFT.md

```diff
@@ -47,6 +53,31 @@ volumes:
       emptyDir: {}
 ```
 
+While it's possible to declare runtime volumes for empty directories, it is not possible for `${APPLICATION_HOME}/conf`. When starting up, Jira and Confluence generate a few configuration files which is a part of image entrypoint. Without anyuid SCC, an unprivileged user can't write to `${APPLICATION_HOME}/conf`. When starting Jira in OpenShift without anyuid SCC attached to jira service account, you will see the following log:
```

Comment:
"of image entrypoint" -> "of the image entrypoint"?

Reply:
Fixed